### PR TITLE
Added docs&help.html page

### PR DESCRIPTION
**Pull request by kefeh**
Started with the docs&help file. Did some few modifications on the _card.scss

### CODE REVIEW

File: data_Insights_website/templates/pages/docs&help.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            <div class="collapse navbar-collapse" id="menu">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item">
+                        <a href="#" class="nav-link hvr-overline-from-center active">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#about" class="nav-link hvr-overline-from-center">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#get-involved" class="nav-link hvr-overline-from-center">Get Involved</a>
```
Comment (AsnelChristian) [inline data_Insights_website/templates/pages/docs&help.html]: elements of the navbar that are not on this page should be removed

File: data_Insights_website/templates/pages/docs&help.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                    </li>
+                    <li class="nav-item">
+                        <a href="#meet-community" class="nav-link hvr-overline-from-center">Our Community</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link hvr-sweep-to-bottom  button-red-sweep-to-bottom button" role=button>Dashboard</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link hvr-sweep-to-top button-blue-sweep-to-top button">Docs &amp; Help</a>
```
Comment (AsnelChristian) [inline data_Insights_website/templates/pages/docs&help.html]: This button should always be filled, so that, one can easily tell where he is on the page just by looking at the header, and maybe it should be aligned at the center

Comment (AsnelChristian) [reply data_Insights_website/templates/pages/docs&help.html]: I mean the center of the navbar

File: data_Insights_website/static/scss/components/_cards.scss [hunk 3]
```diff
@@ -8,10 +8,11 @@
     color: white;
 }
 
-@mixin generate-cards ($name, $color, $width, $height, $img-half:false) {
+@mixin generate-cards ($name, $bg-color, $color, $width, $height, $img-half:false) {
```
Comment (AsnelChristian) [inline data_Insights_website/static/scss/components/_cards.scss]: this should be in a different commit

File: data_Insights_website/static/scss/components/_cards.scss [hunk 4]
```diff
@@ -8,10 +8,11 @@
...
     .card-#{$name} {
         @extend %card;
-        background-color: $color;
+        background-color: $bg-color;
```
Comment (AsnelChristian) [inline data_Insights_website/static/scss/components/_cards.scss]: ^

File: data_Insights_website/static/scss/components/_cards.scss [hunk 5]
```diff
@@ -8,18 +8,17 @@
...
-@mixin generate-cards ($name, $bg-color, $color, $width, $height, $img-half:false) {
+@mixin generate-cards ($name, $bg-color, $width, $height, $img-half:false) {
     .card-#{$name} {
         @extend %card;
         background-color: $bg-color;
-        color: $color;
         width: $width;
         height: $height;
-        margin: 0 35px;
+        margin: 0 4rem;
```
Comment (AsnelChristian) [inline data_Insights_website/static/scss/components/_cards.scss]: this value should be computed as discussed

File: data_Insights_website/static/scss/components/_cards.scss [hunk 6]
```diff
@@ -8,18 +8,17 @@
...
         display: inline-block;
         .card-#{$name}-text {
             font-family: $font-stack-quicksand;
-            padding: 30px 20px;
+            padding: 1rem 2rem;
```
Comment (AsnelChristian) [inline data_Insights_website/static/scss/components/_cards.scss]: ^

File: data_Insights_website/static/css/style.css [hunk 7]
```diff
@@ -1,331 +1,301 @@
...
-    bottom: -35px;
-}
+  bottom: -35px; }
 
 #quotes-carousel {
-    width: 750px;
-    margin: auto;
-}
+  width: 750px;
+  margin: auto; }
```
Comment (AsnelChristian) [inline data_Insights_website/static/css/style.css]: You forgot to run beautify this css file, so this commit is coming to undo the previous one

Comment (AsnelChristian) [reply data_Insights_website/static/css/style.css]: You should probably delete the previous commit and create a new one where you format this file appropriately

File: data_Insights_website/static/css/style.css [hunk 8]
```diff
@@ -1,301 +1,449 @@
...
-  bottom: -35px; }
+    bottom: -35px;
+}
 
 #quotes-carousel {
-  width: 750px;
-  margin: auto; }
+    width: 750px;
+    margin: auto;
+}
```
Comment (AsnelChristian) [inline data_Insights_website/static/css/style.css]: You should delete this commit and simply add one at the end that does this when you are done with all the modifications, you might end up with the same issues as in the previous commit if you do it like this

File: data_Insights_website/static/scss/components/_cards.scss [hunk 9]
```diff
@@ -3,11 +3,18 @@
...
+.lg-card-header {
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+}
 %card{
     text-align: center;
     color: white;
 }
-
+%container{
```
Comment (AsnelChristian) [inline data_Insights_website/static/scss/components/_cards.scss]: move thinks concerning div to _row.scss (maybe we should rename this file)

**Comment by AsnelChristian** [comment]
Please don't use pass tense in commit short log, and also mention the name of the file you are modifying in your commit

File: data_Insights_website/templates/pages/docs&help.html (hunk 1 above)
Comment (kefeh) [reply data_Insights_website/templates/pages/docs&help.html]: Done

**Comment by AsnelChristian** [comment]
@rultor

**Comment by AsnelChristian** [comment]
@rultor merge

**Comment by kefeh** [comment]
Hello @AsnelChristian can you please grant @rultor enough access rights so that this pull request should be merged in time for continuity.
